src/software/physics/euclidean_to_wheel.m

```diff
@@ -0,0 +1,56 @@
+function target_wheel_speeds = euclidean_to_wheel(current_wheel_speeds, target_euclidean_velocity)
+%EUCLIDEAN_TO_WHEEL Converts Euclidean velocity to robot wheel speeds.
```

Comment:
curious what this file is all about

Reply:
This was for cross checking the formulas, just did it through matlab.